integ-test/src/test/java/org/opensearch/sql/ppl/WildcardQueryIT.java

```diff
@@ -0,0 +1,145 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.sql.ppl;
+
+import static org.opensearch.sql.legacy.TestsConstants.TEST_INDEX_WILDCARD;
+import static org.opensearch.sql.util.MatcherUtils.rows;
+import static org.opensearch.sql.util.MatcherUtils.verifyDataRows;
+
+import java.io.IOException;
+import org.json.JSONObject;
+import org.junit.Test;
+
+public class WildcardQueryIT extends PPLIntegTestCase {
+
+  @Override
+  protected void init() throws Exception {
+    loadIndex(Index.WILDCARD);
+  }
+
+  @Test
+  public void test_wildcard_query_asterisk_function() throws IOException {
+    String expected = "test wildcard";
+
+    String query = "source=" + TEST_INDEX_WILDCARD + " | where wildcard_query(Body, 't*') | head 1";
+    JSONObject result = executeQuery(query);
+    verifyDataRows(result, rows(expected));
+  }
+
+  @Test
+  public void test_wildcard_query_question_mark_function() throws IOException {
+    String expected = "test wildcard";
+
+    String query = "source=" + TEST_INDEX_WILDCARD + " | where wildcard_query(Body, 'test wild??rd') | head 1";
+    JSONObject result = executeQuery(query);
+    verifyDataRows(result, rows(expected));
+  }
+
+  //  SQL uses ? as a wildcard which is converted to * in WildcardQuery.java
+  @Test
+  public void test_wildcard_query_sql_wildcard_percent_conversion() throws IOException {
```

Comment:
Should we test that the output of a query using a `%` is the same as one with a `*`? To me this test just looks like it verifies we get an expected result, but I think we should also test the outputs against each other to make sure that they are behaving the same way.

Reply:
Yes. I will add that. It was already testing it before but was removed in the process of redoing the test cases. Thanks!

Reply:
Added in 1101ca377e29f1b53752d43d32399abb01c7abd6